CycleOpsFluid2PowerCalculator2.py

```diff
@@ -0,0 +1,48 @@
+from AbstractPowerCalculator import AbstractPowerCalculator
```

Comment:
This file, ending in "2", seems to be surplus to requirements

Reply:
Unfortunately I can only approve the entire pull request. Maybe create a new branch at your last commit, and then reset the original branch to the correct commit and push that to the PR branch?

Reply:
If that's all too complicated, just close this PR, create a new branch at the correct commit, and push that as a new PR.